src/cascadia/TerminalApp/TerminalWindow.idl

```diff
@@ -136,5 +136,8 @@ namespace TerminalApp
 
         event Windows.Foundation.TypedEventHandler<Object, SettingsLoadEventArgs> SettingsChanged;
 
+        event Windows.Foundation.TypedEventHandler<Object, RequestMoveContentArgs> RequestMoveContent;
+        void AttachContent(String content, UInt32 tabIndex);

```

Comment:
Somebody should be able to call this on the (effectively) public API surface of TerminalWindow?